documentation/api/query/v4/operators.markdown

```diff
@@ -204,6 +204,38 @@ When querying a particular endpoint, such as `/pdb/query/v4/nodes` the endpoint
 the context for the query. While querying the [root] endpoint requires specifying a
 context explicitly.
 
+## Paging Operators (`limit`, `offset`, `order_by`)
+
+PuppetDB allows specification of paging clauses within a "from" clause in a
+query or subquery. The `limit` and `offset` operators both accept an
+integer-valued argument, and `order_by` accepts a vector of either column names
+or vector pairs containing a column name and an ordering of "asc" or "desc".
+For example,
+
+    ["order_by", ["certname", ["timestamp", "desc"]]]
```

Comment:
Breaking down this paragraph above, showing an example like this for each might be nice. Like "The limit and offset operators both accept an integer valued argument: <insert examples>. The 'order_by' ... blah blah: <insert examples>"